example/Src/main.c

```diff
@@ -413,6 +413,61 @@ void HAL_GPIO_EXTI_Callback(uint16_t GPIO_Pin)
     	}
     }
 }
+
+/*@brief this fuction does One-Shot for MCP3564 */
+void ADC_One_Shot(SPI_HandleTypeDef *hspi)
+{
+ int status;
+ int notReady;
+
+ // Pull chip select low
+ HAL_GPIO_WritePin(SPI1_CS_GPIO_Port, SPI1_CS_Pin, 0);
+
+ // Write command byte and get status in return
+ uint8_t cmd;
+ cmd = readConversionData;
+ HAL_SPI_TransmitReceive(hspi, &cmd,0, 1, 10);
+
+ // Check the status byte for the POR bit. Low = POR has occurred
+ if ((status & 0x01) == 0x00)
+ {
+	 //Use your own debug mode
+	 //Lcd_cursor(&lcd,0,9);

```

Comment:
Remove code fragments regarding LCD

Reply:
> Yo, thanks! My first code review. So thanks for the opportunity. And of course thanks for the debugging work 😃 As you can see, I can't just merge it but incorporate your input a little differently later.

Thanks! i will implement you suggestions